### DISCUSSION TITLE
What does specifying the crate type as "cdylib" and "rlib" mean in a `cargo-leptos` project?

### DISCUSSION THREAD

**Discussion by saikatdas0790** [leptos-rs/leptos#1961] [Q&A] [answered]
I mean this:
![image](https://github.com/leptos-rs/leptos/assets/7412443/f0e0dba0-954e-49e4-a365-eeb7bbe6e2b3)

I ask this because when I build the project for a musl target, I get this warning. Is this an issue?
![image](https://github.com/leptos-rs/leptos/assets/7412443/86dfd999-c6fd-4929-ba5a-a1ed39848644)

If I build the server for a `musl` target and deploy that alongside the `site` directory that `cargo-leptos` build, it should just work, right?

**Comment by gbj** [comment, marked as answer]
I think these two things are unrelated. `cargo-leptos` doesn't have a `--target` argument, which is the error. I don't think the warning is related.

The crate is marked as `cdylib` because this is the crate type used for building an app to WASM, generally.

I think I just answered the `--target` part in [my reply to your post about that](https://github.com/leptos-rs/leptos/discussions/1963#discussioncomment-7417136), which I'm linking here for anyone searching in the future.
